mkdocs.yml

```diff
@@ -8802,8 +8802,8 @@ nav:
   - gama format: agama/gama-format.md
 - 'Jans Casa':
     - 'casa/index.md'
-    - Quick Start: 'casa/quick-start.md'
     - Administration Guide:
+        - 'Quick Start': 'casa/administration/quick-start.md'
```

Comment:
@jgomer2001 since there is a change in the location of the quick-start guide, please also check if there are any references to quick-start guide from other documents. Update those references to point to new path.

Reply:
yes, I checked this already